admin/orders.php

```diff
@@ -173,57 +179,57 @@
         // adjust download_maxdays based on current date
 
         // check for existing product attribute download days and max
-        $chk_products_download_query = "SELECT orders_products_id, orders_products_filename, products_prid
-                                        FROM " . TABLE_ORDERS_PRODUCTS_DOWNLOAD . "
-                                        WHERE orders_products_download_id = " . (int)$_GET['download_reset_on'];
+        $chk_products_download_query = 'SELECT orders_products_id, orders_products_filename, products_prid
+                                        FROM ' . TABLE_ORDERS_PRODUCTS_DOWNLOAD . '
+                                        WHERE orders_products_download_id = ' . (int)$_GET['download_reset_on'];
         $chk_products_download = $db->Execute($chk_products_download_query);
 
-        $chk_products_download_time_query = "SELECT pa.products_attributes_id, pa.products_id,
+        $chk_products_download_time_query = 'SELECT pa.products_attributes_id, pa.products_id,
                                                     pad.products_attributes_filename, pad.products_attributes_maxdays, pad.products_attributes_maxcount
-                                             FROM " . TABLE_PRODUCTS_ATTRIBUTES . " pa,
-                                                   " . TABLE_PRODUCTS_ATTRIBUTES_DOWNLOAD . " pad
+                                             FROM ' . TABLE_PRODUCTS_ATTRIBUTES . ' pa,
+                                                   ' . TABLE_PRODUCTS_ATTRIBUTES_DOWNLOAD . " pad
                                              WHERE pa.products_attributes_id = pad.products_attributes_id
                                              AND pad.products_attributes_filename = '" . $db->prepare_input($chk_products_download->fields['orders_products_filename']) . "'
                                              AND pa.products_id = " . (int)$chk_products_download->fields['products_prid'];
 
         $chk_products_download_time = $db->Execute($chk_products_download_time_query);
```

Comment:
This may be the desire, but there is not unique information in this query that forces/mandates results be a single row response. 

Not modifying this one other than to change to using `INNER JOIN` style.

I can see a potential argument that a given product would not likely have two downloads having the same filename or some other argument; however, I am not seeing that such is a requirement, just a method of implementation.